src/sass/variables/_Icon.Variables.scss

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
 
 // Icon directory and paths
-$ms-icon-cdn-path: 'https://static2.sharepointonline.com/files/fabric/assets/icons' !default;
+$ms-cdn-build-date: '20221201.001';
+$ms-cdn-path: 'https://res-1.cdn.office.net/files/fabric-cdn-prod_#{$ms-cdn-build-date}/assets' !default;
```

Comment:
lets move these out to another file. i think this one should only be about icons in the same way that BrandIcons.Variables is only about BrandIcons

maybe keep it simple with a _cdn.variables.scss file